crates/rooch-key/src/keystore/account_keystore.rs

```diff
@@ -81,17 +79,16 @@ pub trait AccountKeystore {
         Ok(mnemonic_phrase)
     }
 
-    fn export_private_key(&mut self, sk_bytes: &[u8]) -> Result<String, anyhow::Error> {
-        // get 33 bytes flag and secret key
-        let mut priv_key_bytes = Vec::with_capacity(sk_bytes.len() + 1);
-        // supports secp256k1 signature scheme
-        priv_key_bytes.push(SignatureScheme::Secp256k1.flag());
-        priv_key_bytes.extend_from_slice(sk_bytes);
-        // init `roochsecretkey` as HRP
-        let hrp = Hrp::parse(ROOCH_SECRET_KEY_PREFIX)?;
-        // encode hrp and 33 bytes private key using bech32 method
-        let bech32_encoded = encode::<Bech32>(hrp, &priv_key_bytes)?;
-        Ok(bech32_encoded)
+    fn import_external_account(
+        &mut self,
+        address: RoochAddress,
+        kp: RoochKeyPair,
+        password: Option<String>,
+    ) -> Result<(), anyhow::Error> {
+        let private_key_encryption = EncryptionData::encrypt_with_type(&kp, password)?;
+        // TODO: whether to add address to mnemonic data? It's either associated with the same mnemonic or not. There needs a check if the imported account is from the same mnemonic phrase.
```

Comment:
The address in mnemonic data is derived from the mnemonic. So we should not add the address to `mnemonic data`.

The address not in mnemonic data is an external account address.